Dockerfile

```diff
@@ -193,33 +194,54 @@ COPY . /work
 WORKDIR /work
 RUN ./pkg/rpm/build.sh
 
-FROM scratch
-COPY --from=buster /tmp/google-cloud-ops-agent.tgz /google-cloud-ops-agent-debian-buster.tgz
-COPY --from=buster /google-cloud-ops-agent*.deb /
+FROM scratch AS buster
```

Comment:
Personal nit (feel free to ignore):

I would find `debian10-buster` clearer than just `buster`.  It would benefit those of us who don't keep a (nickname -> distro) map in main memory, and also make the list at 238-247 clearer.

Reply:
This is a backward-compatible change — the target was already called `buster` (now `buster-build`), I'm just reusing the name for another target. We can discuss renaming the targets in a follow-up PR.